src/components/Confirm/index.js

```diff
@@ -22,13 +22,13 @@ const FileViewer = ({file:{preview, type}}) =>
   </object>
 
 
-const Capture = ({capture:{file}}) =>
-  <div className={style.captures}>
-    {isOfFileType(['pdf'], file) ?
-      <FileViewer file={file}/> :
```

Comment:
I still think we need to keep pdf. We are not converting it to an image and some browser won't support pdf (in base64 format) on the image tag.
I could be wrong though.